src/Tribe/Linked_Posts/Chooser_Meta_Box.php

```diff
@@ -186,23 +186,31 @@ protected function single_post_dropdown( $linked_post_id ) {
 	/**
 	 * Render a link to edit the linked post
 	 *
+	 * @since TBD Reversed check for editing posts. Added check if the $edit_link is empty.
 	 * @since 3.0
 	 *
 	 * @param int $linked_post_id
 	 */
 	protected function edit_post_link( $linked_post_id ) {
 		$linked_post_pto = get_post_type_object( $this->post_type );
-		$cannot_edit_others_posts = (
-			empty( $linked_post_pto->cap->edit_others_posts )
-			|| ! current_user_can( $linked_post_pto->cap->edit_others_posts )
+
+		$can_edit_others_posts = (
+				! empty( $linked_post_pto->cap->edit_others_posts )
+				&& current_user_can( $linked_post_pto->cap->edit_others_posts )
 		);
 
-		if ( is_admin() && $cannot_edit_others_posts ) {
+		// Bail if you are unable to edit the type of post.
+		if ( ! $can_edit_others_posts ) {
 			return;
 		}
 
 		$edit_link = get_edit_post_link( $linked_post_id );
 
+		// Bail if $edit_link is empty.
+		if ( empty( $edit_link ) ) {
+			return;
+		}
```

Comment:
Might be more succinct to just do:
```php
// Bail if the user is unable to edit the type of post.
if (
    empty( $linked_post_pto->cap->edit_others_posts )
    || ! current_user_can( $linked_post_pto->cap->edit_others_posts )
) {
    return;
}
```

Reply:
Sorry - meant for this to go on the `if ( ! $can_edit_others_posts ) {` line (203) 🤦🏼

Reply:
Good call on simplifying the logic. I just made the change :+1: